integration_test/tests/wallet.rs

```diff
@@ -74,13 +74,25 @@ pub fn get_addresses_by_label() {
 }
 
 #[test]
-// TODO: Consider testing a few different address types.
 #[cfg(feature = "TODO")]
+// TODO: We are getting an error for missing `pubkeys` field but the field is optional?
```

Comment:
In 86dc357af3eb900e1e606776a3490a4bff3bb802:

You need `#[serde(default)]` to make an `Option<T>` field actually optional. Otherwise it is required to be present and allowed to be `null`.

Reply:
Legend. So that means all the other `Option` fields must be getting `null` because this is the first time I had this problem and its definitely not the first usage of `Option`. TIL. Thanks.